### ISSUE TITLE
1 Question per screen

### ISSUE THREAD

**Issue by jordantomax** [AdmitHub/college-chooser#1]
Some of the questions within the bot actually include 2 questions, such as "when do you expect to graduate" and "what is your dream college".

For the sake of having the user's focus, we should make these 2 questions.

**Comment by AndrewMagliozzi** [comment]
I think Charlie was just using these to show us how to put multiple
elements into a single question.

**Comment by yourcelf** [comment]
Yup -- it's easy for us to configure with the given stuff.  The FTB implementation right now is not what I expect us to continue to use as the initial questionaire; that remains to be developed.  But the FTB does show the full breadth of question types.

Design-wise: I'll leave it up to you both whether you'd like to always have one question per page to simplify the style implementation, or to design with the expectation that some future questionnaires will have more than one question per page.

**Comment by jordantomax** [comment]
Ah I see, well it's okay to have multiple fields, but we should probably avoid having multiple headers at the very least.

**Comment by yourcelf** [comment]
Jordan, if it helps, check out https://github.com/AdmitHub/college-chooser/blob/master/lib/formflows/freetrialbot.js.  This is the format for authoring the full flow of a questionnaire.  That gets turned into markup via the templates in https://github.com/AdmitHub/college-chooser/tree/master/client/autoform.

You'll notice the "prompt" field in freetrialbot.js is probably the thing that results in headers; it can be added/removed from the flow easily.  I recognize the markup gets a little noodly with some of the nested autoform fields but I hope it's not too opaque.

**Comment by jordantomax** [comment]
Sweet, thanks Charlie

**Comment by jordantomax** [comment]
Hmm, I'm having some difficulty with this. In freeTrialBot.js, I see questions that ask "What year do you expect to graduate from high school?", but in the browser i see "When do you expect to graduate from high school?" Also, I don't see any indication that questions 2 and 3 have been combined. What am I missing?

**Comment by yourcelf** [comment]
Make sure you're not confusing ``lib/formflows/freetrialbot.js`` with ``server/workflows/freeTrialBot.js``.

Workflows == SMS-based single-question-at-a-time back and forth.  In meteor, it's server-side only, because all interaction happens through SMS and the twilio API.

Formflows == web-based form wizard.

I split them out because the optimal UI for each is rather different; and the current architecture makes it lightweight to offer a script for both.  Sorry about the naming, it's obtuse; if you have suggestions for less confusing naming I'd love to hear.

**Comment by jordantomax** [comment]
aha, thank you sir.
